web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use Dashboard Upstream if:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
+
+### Installing using Dashboard Upstream
+- Create an empty upstream site:
+
+#### Via the Pantheon Dashboard at this link: 
+- https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65
+
+#### Or Alternatively via Terminus:
+
+```
+terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+```
+
+Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+#### In the Pantheon Admin Dashboard:
+
+Switch **Development Mode** to **Git**
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**. 
```

Comment:
I think our recommendation needs to be a little different for the general public. We should recommend that they upgrade from a sandbox site, and suggest that they select the plan that is appropriate for their site.